### PR TITLE
Restyle empty state page and add instructions

### PR DESCRIPTION
**Pull request by CtrlSpice**
empty-state-instructions

### CODE REVIEW

**Review by jmorrell — APPROVED** [review]
DALL·E 2023-02-08 10 46 31 - a corgi reading instructions, putting together ikea furniture, kids cartoon

File: README.md [hunk 1]
```diff
@@ -118,11 +118,11 @@ data, but when working locally our experience often looks more like this:
...
 You can use [Jaeger's all-in-one](https://www.jaegertracing.io/docs/1.41/deployment/#all-in-one)
-distribution, but this requires quite a bit of additional knowledge for the end-user of docker,
-configuration options, and the user experience is not focused around "show me the data I just
-emitted".
+distribution, but this requires quite a bit of additional knowledge for the end-user around docker and
+navigating a lot of configuration options. Additionally the user experience is not focused around 
+"show me the data I just emitted".
 
-The goal for `otel-desktop-viewer` is to allow a user to install it with one command, require
+The goals for `otel-desktop-viewer` are to allow a user to install it with one command, require
```
[the remark this answers is not in the document]
Comment (CtrlSpice) [reply README.md:125]: oops I will try to fix it

**Comment by CtrlSpice** [comment]
final-lulu

File: desktop-exporter/app/components/empty-state-view/empty-state-view.tsx [hunk 2]
```diff
@@ -72,67 +71,200 @@ async function pollTraceCount() {
...
-          />
-          <Text size="sm">Artwork credit goes here</Text>
+        <CardHeader
+          borderRadius="lg"
+          paddingY="0"
+        >
+          <Flex width="100%">
+            <Image
+              src="assets/images/lulu.png"
+              alt="A pink axolotl is striking a heroic pose while gazing at a field of stars through a telescope. Her name is Lulu Axol'Otel the First, valiant adventurer and observability queen."
```
Comment (jmorrell) [inline desktop-exporter/app/components/empty-state-view/empty-state-view.tsx:105]: ❤️

**Review by jmorrell — APPROVED** [review]
DALL·E 2023-02-08 10 48 23 - a corgi reading instructions, putting together ikea furniture, kids cartoon

Much improved!

File: desktop-exporter/app/components/empty-state-view/empty-state-view.tsx (hunk 2 above)
Comment (CtrlSpice) [reply desktop-exporter/app/components/empty-state-view/empty-state-view.tsx:105]: I love Lulu a lotl!

**Pull request merged**